### PR TITLE
#250 Feature:implemented a footer

### PR DESCRIPTION
**Pull request by shreyanka155**
### Implemented a footer for creating social icons.

(Write your answer here.)

### Checklist

- [x] I am making a proper pull request, not spam.
- [x] I've checked the issue list before deciding what to submit.

Screenshot (6)

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -505,16 +505,34 @@ h1 {
...
+.icons {
+  padding: 1rem;
+
 
-.foot a {
-  color: #737373;
 }
+.icons i{
+  padding: 0.5rem;
+  background-color:#403e3e;
```
Comment (wasimreja) [inline style.css:526]:
```suggestion
  background-color: #33353d;
```

File: style.css [hunk 2]
```diff
@@ -505,16 +505,34 @@ h1 {
...
+  border-radius: 50%;
+  margin: auto 0.2rem;
+  color:#737373;
```
Comment (wasimreja) [inline style.css:529]:
```suggestion
  color: #818a91;
```

**Review by wasimreja — CHANGES_REQUESTED** [review]
Add hover to icons.

Twitter background color: #00aced
LinkedIn background color: #007bb6
Instagram background color: #ea2c59 

Add `transition: all .2s linear`

**Comment by shreyanka155** [comment]
Which width and height do i change....icons or background?

**Comment by shreyanka155** [comment]
Please review my code again i have updated the code using "amend" in the same commit.

**Comment by wasimreja** [comment]
@shreyanka155 all good but the icons background color doesn't change on hover

**Comment by shreyanka155** [comment]
Will fix the issue within today .....

**Comment by shreyanka155** [comment]
Fixed the issue,please check again.Let me know if further changes required.

**Review by wasimreja — CHANGES_REQUESTED** [review]
remove padding instead add this
```
.icons i {
  width: 40px;
  height: 40px;
  line-height: 40px;
  margin-left: 6px;
  border-radius: 100%;
  background-color: #33353d;
  color: #818a91;
  display: inline-block;
  text-align: center;
  transition: all .2s linear;
```

add white color to icons on hover
```
.twitter :hover {
  background-color: #00aced;
  color: #fff;
}

.linkedin :hover{
  background-color: #007bb6;
  color: #fff;
}

.instagram :hover{
  background-color: #ea2c59;
  color: #fff;
}

File: style.css [hunk 3]
```diff
@@ -507,16 +507,51 @@ h1 {
...
 .foot {
 	color: #737373;
-	text-align: center;
 	font-family: "Comfortaa", cursive;
-	padding: 2rem 2rem;
-	bottom: 0%;
-	width: 100%;
+  display: flex;
+  justify-content: center;
+  color: #e3e2e9;
```
Comment (wasimreja) [inline style.css:513]: remove this
```suggestion
```

**Comment by shreyanka155** [comment]
Code updated and colour removed.Please check.

**Pull request merged**